public/app/plugins/datasource/mssql/partials/config.html

```diff
@@ -27,6 +27,22 @@ <h3 class="page-heading">MSSQL Connection</h3>
 			<a class="btn btn-secondary gf-form-btn" href="#" ng-click="ctrl.current.secureJsonFields.password = false">reset</a>
 		</div>
 	</div>
+
+	<div class="gf-form">
+		<label class="gf-form-label width-7">Encrypt</label>
+		<div class="gf-form-select-wrapper max-width-15 gf-form-select-wrapper--has-help-icon">
+			<select class="gf-form-input" ng-model="ctrl.current.jsondata.sslmode" ng-options="mode for mode in ['disable', 'false', 'true']" ng-init="ctrl.current.jsondata.sslmode"></select>
```

Comment:
On second thought I think we should rename the `sslmode` to `encrypt` so that it's not mixed up with the sslmode setting for postgres. Also because we've named it Encrypt in the UI.

Also, for existing MSSQL datasources the default `false` is not set in encrypt dropdown:

![image](https://user-images.githubusercontent.com/1668778/46982282-f3b5e700-d0db-11e8-8635-73acc45cea7f.png)

Think you need to set default in config_ctrl if not defined, like postgres have done [here](https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/postgres/config_ctrl.ts#L13). However, MSSQL doesn't have a config_ctrl.ts, only a class defined [here](https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/mssql/module.ts#L4), so you'll have to move that to a separate config_ctrl.ts file, similar to postgres, and implement the default handling in the constructor.